modules/ols-disabling-data-collection-cluster.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-disabling-data-collection-cluster_{context}"]
+= Disabling data collection on a cluster
+
+Delete the pull secret on an {ocp-short-name} cluster to disable data collection for the cluster. Disable data collection on an {ocp-short-name} cluster to protect sensitive information, ensure compliance, or address privacy concerns.
+
+.Prerequisites
+
+* You have access to the {oc-first} and are logged in as a user with the `cluster-admin` role. 
+
+* You have an LLM provider available for use with the {ols-long} Service.
+
+* You have installed the {ols-long} Operator.
+
+* You have deployed the {ols-long} Service.
+
+.Procedure
+
+. Confirm that remote data collection is enabled for the cluster and inspect the specific credentials used by running the following command:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' | jq '.auths."cloud.openshift.com"'
+----
+
+. Retrieves the raw, Base64-encoded contents of the global pull-secret for the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config
+----
+
+. Delete the pull secret by running the following command:
```

Comment:
Deleting the whole pull-secret is not a good idea. Do we need to duplicate this information about opting out from telemetry? It is here already https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/support/remote-health-monitoring-with-connected-clusters#opting-out-remote-health-reporting